src/components/SearchBox.tsx

```diff
@@ -150,6 +176,15 @@ const SearchBox = ({ setSearchResults }: SearchBoxProps) => {
         >
           Clear
         </Button>
+        <Button
+          onClick={getRandomArtist}
+          variant='contained'
+          color='secondary'
+          size='large'
+          aria-label='random-artist'
+        >
+          I am one lucky onion
```

Comment:
Maybe more subtle approach to the button name like "Random" or something like that 🙂

Reply:
agreed, this was just a concept :smile: